dubbo-configcenter/dubbo-configcenter-api/src/main/java/org/apache/dubbo/configcenter/DynamicConfiguration.java

```diff
@@ -104,6 +111,22 @@ default String getConfig(String key, String group) {
      */
     String getConfig(String key, String group, long timeout) throws IllegalStateException;
 
+    /**
+     * {@see #getConfig(String, String, long)}
+     *
+     * This method are mostly used to get a compound config file, such as a complete dubbo.properties file.
+     */
+    default String getConfigFile(String key, String group) throws IllegalStateException {
+        return getConfigFile(key, group, 0L);
+    }
+
+    /**
+     * {@see #getConfig(String, String, long)}
+     *
+     * This method are mostly used to get a compound config file, such as a complete dubbo.properties file.
+     */
+    String getConfigFile(String key, String group, long timeout) throws IllegalStateException;
```

Comment:
As this new overloaded method has added with timeout, looking at related changes I not seeing any one is using **timeout**. Are we going to use it in recent future ?